dae/dae/genomic_resources/cli.py

```diff
@@ -113,12 +117,30 @@ def _configure_list_subparser(subparsers):
 
 def _run_list_command(
         proto: Union[ReadOnlyRepositoryProtocol, GenomicResourceRepo], _args):
-    for res in proto.get_all_resources():
-        res_size = sum(fs for _, fs in res.get_manifest().get_files())
-        print(
-            f"{res.get_type():20} {res.get_version_str():7s} "
-            f"{len(list(res.get_manifest().get_files())):2d} {res_size:12d} "
-            f"{res.get_id()}", file=sys.stderr)
+    protocols = []
+
+    def is_group_repo(repo):
+        if type(repo) is GenomicResourceGroupRepo:
+            for child in repo.children:
+                is_group_repo(child)
+        else:
+            protocols.append(repo)
+
+    is_group_repo(proto)
```

Comment:
`is_group_repo` is an ambiguous name for 2 reasons, the function appends values to a list and doesn't return True/False and it also handles repositories and protocols. I suggest moving the type checking outside of the function and keeping the function simple - to recursively collect all repositories in a group into a list.